src/Components/Artwork/ShelfArtwork.tsx

```diff
@@ -41,58 +44,61 @@ const ShelfArtwork: React.FC<ShelfArtworkProps> = ({
     (artwork.artistNames ? ` by ${artwork.artistNames}` : "")
 
   return (
-    <RouterLink
-      to={artwork?.href}
-      onClick={onClick}
-      onMouseEnter={onMouseEnter}
-      onMouseLeave={onMouseLeave}
-      display="flex"
-      flexDirection="column"
-      justifyContent="flex-end"
-      textDecoration="none"
-      data-test="artworkShelfArtwork"
-      data-testid="ShelfArtwork"
-      aria-label={label}
-      width={width}
-      {...rest}
-    >
-      {image ? (
-        <Box
-          maxHeight={[250, 320]}
-          maxWidth="100%"
-          style={{
-            aspectRatio: `${artwork.image?.width ?? 1} / ${
-              artwork.image?.height ?? 1
-            }`,
-          }}
-          bg="black10"
-        >
-          <Image
-            src={image.src}
-            srcSet={image.srcSet}
-            width="100%"
-            height="100%"
-            lazyLoad={lazyLoad}
-            style={{ objectFit: "contain" }}
-            alt=""
-          />
-        </Box>
-      ) : (
-        <Box style={{ aspectRatio: "1 / 1" }} maxWidth="100%" bg="black10" />
-      )}
+    <ManageArtworkForSavesProvider savedListId={savedListId}>
+      <RouterLink
+        to={artwork?.href}
+        onClick={onClick}
+        onMouseEnter={onMouseEnter}
+        onMouseLeave={onMouseLeave}
+        display="flex"
+        flexDirection="column"
+        justifyContent="flex-end"
+        textDecoration="none"
+        data-test="artworkShelfArtwork"
+        data-testid="ShelfArtwork"
+        aria-label={label}
+        width={width}
+        {...rest}
+      >
+        {image ? (
+          <Box
+            maxHeight={[250, 320]}
+            maxWidth="100%"
+            style={{
+              aspectRatio: `${artwork.image?.width ?? 1} / ${
+                artwork.image?.height ?? 1
+              }`,
+            }}
+            bg="black10"
+          >
+            <Image
+              src={image.src}
+              srcSet={image.srcSet}
+              width="100%"
+              height="100%"
+              lazyLoad={lazyLoad}
+              style={{ objectFit: "contain" }}
+              alt=""
+            />
+          </Box>
+        ) : (
+          <Box style={{ aspectRatio: "1 / 1" }} maxWidth="100%" bg="black10" />
+        )}
 
-      <Metadata
-        artwork={artwork}
-        hideSaleInfo={hideSaleInfo}
-        isHovered={isHovered}
-        contextModule={contextModule}
-        showSaveButton
-        disableRouterLinking
-        maxWidth="100%"
-      />
+        <Metadata
+          artwork={artwork}
+          hideSaleInfo={hideSaleInfo}
+          isHovered={isHovered}
+          contextModule={contextModule}
+          showSaveButton
+          disableRouterLinking
+          maxWidth="100%"
+          enableSaveButtonForLists={!artwork.isInAuction}
```

Comment:
excludes in auction works from the flow (for artwork rails)

Reply:
**question**: maybe it makes sense to do [this check](https://github.com/artsy/force/blob/1716d72df521eacea2af9a32eeae8344dc17d8af/src/Components/Artwork/Details.tsx#L231) in the `Details` component itself, since now the new behavior of the 💙 button will be on all artwork grids/rails? Also we can safely remove `enableSaveButtonForLists` since there is no need for this prop anymore

Reply:
hmm, good catch. perhaps checking `isAuctionArtwork` will be enough for our case

cc: @damassi

Reply:
I think it is better to check `isInAuction` (as it will be safer)

Reply:
amazing, resolving this convo

Reply:
[Just for context] Since `isAuctionArtwork` will not always be `true` if the artwork is in an auction. For example, as in the attached video (if `isAuctionArtwork` is `true`, then there should be a **red** background. But in our example, the background is green even though the artwork is at auction)

Reply:
**question (non-blocking):**  Does `isAuctionArtwork` still have value then, or should it be deprecated?